packages/database/index.ts

```diff
@@ -32,6 +32,8 @@ import { setSDKVersion } from './src/core/version';
 
 const ServerValue = Database.ServerValue;
 
+export { FIREBASE_DATABASE_EMULATOR_HOST_VAR } from './src/core/RepoManager';
```

Comment:
Why are we exporting this here?  Are you hoping to make this part of the public API?  I'm not sure that's desirable...  If we do want to make it part of the public API then it needs to be added to our .d.ts files and we would need to go through an internal API review process...

Reply:
Perhaps it shouldn't be exposed. The intended use case is to be able to set this variable from the firebase nodejs admin sdk and I figured having a single source of truth might've been useful, but we can also just define the variable once in each repo and keep them in sync.

Reply:
Oh!  I see.  That actually is probably fine.  How about just adding a comment then?

Reply:
I don't think it's worth putting this change up through API review. I will define the variables separately and add a cross-referencing comment to both repos.